components/footer.php

```diff
@@ -54,6 +54,11 @@
 							<i class="bi-twitter fs-3"></i>
 						</a>
 					</li>
+					<li class="list-inline-item me-4">
+						<a rel="me" href="https://mastodon.social/@miraheze" title="<?php echo getTranslation( 'footer-mastodon-title' ); ?>">
```

Comment:
I think rel="me" should be removed for consistency with others.

Reply:
I put it there because that allows the link on our profile to show as verified.

Reply:
Ah, okay.